src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,34 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    position: relative;
+  }
+
+  &__item {
+    margin-left: 20px;
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    color: $main-text-color;
+    white-space: nowrap;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      color: $main-color;
```

Comment:
The hover color for navigation links should be `#00acdc` as specified in the task requirements. Please update this to meet the task criteria.